buildspec.yml

```diff
@@ -33,3 +33,4 @@ artifacts:
   - 'nessie/**/*'
   - 'requirements.txt'
   - 'run.py'
+  - 'scripts/**/*'
```

Comment:
`scripts/**/*` is not currently required but, eventually, we might want it bundled with deployments.